recipe/meta.yaml

```diff
@@ -45,15 +36,34 @@ requirements:
 test:
   imports:
     - intake
-    - intake.auth
     - intake.catalog
+    - intake.catalog.entry
+    - intake.catalog.local
+    - intake.container
+    - intake.interface
+    - intake.interface.catalog
+    - intake.interface.source
+    - intake.readers
+    - intake.readers.catalogs
+    - intake.readers.convert
+    - intake.readers.datatypes
+    - intake.readers.entry
+    - intake.readers.importlist
+    - intake.readers.metadata
+    - intake.readers.mixins
+    - intake.readers.namespaces
+    - intake.readers.output
+    - intake.readers.readers
+    - intake.readers.search
+    - intake.readers.transform
+    - intake.readers.user_parameters
     - intake.source
+    - intake.source.derived
   requires:
     - pip
+    - packaging
```

Comment:
Why do you have to add packaging here? If it is a pip check error, it may indicate a package was configured wrongly and is missing packaging dependency. It is is due to an import error, it may be that intake forgot to declare it as a run dependency.

Reply:
`Traceback (most recent call last):
  File "/Users/jaidarice/miniconda3/conda-bld/intake_1726012642633/test_tmp/run_test.py", line 17, in <module>
    import intake.interface
  File "/Users/jaidarice/miniconda3/conda-bld/intake_1726012642633/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.11/site-packages/intake/interface/__init__.py", line 7, in <module>
    from packaging.version import Version
ModuleNotFoundError: No module named 'packaging'
import: 'intake'
import: 'intake.catalog'
import: 'intake.catalog.entry'
import: 'intake.catalog.local'
import: 'intake.container'
import: 'intake.interface'
WARNING: Tests failed for intake-2.0.7-py311hca03da5_0.tar.bz2 - moving package to /Users/jaidarice/miniconda3/conda-bld/broken
TESTS FAILED: intake-2.0.7-py311hca03da5_0.tar.bz2`

Reply:
I've added it as a run dep

Reply:
packaging was added in v0.6.7 https://github.com/intake/intake/commit/00d046aa989f2c3697376d975030149276aff874 but we didn't test all imports.
@Jrice1317 Can you comment that it's used here https://github.com/intake/intake/blob/2.0.7/intake/interface/__init__.py#L7? It will prevent us in the future from investigating it again.